tests/Wallabag/ApiBundle/Controller/EntryRestControllerTest.php

```diff
@@ -426,13 +426,6 @@ public function testGetTaggedEntries()
         $this->assertSame('application/json', $this->client->getResponse()->headers->get('Content-Type'));
     }
 
-    public function testGetTaggedEntriesWithBadParams()
-    {
-        $this->client->request('GET', '/api/entries', ['tags' => ['foo', 'bar']]);
-
-        $this->assertSame(200, $this->client->getResponse()->getStatusCode());
-    }
```

Comment:
I think the Symfony BadRequestException is enough instead of handling it ourselves.